stats/opentelemetry/experimental/grpc_trace_bin_propagator.go

```diff
@@ -16,7 +16,12 @@
  *
  */
 
-package opentelemetry
+// # Experimental
```

Comment:
@dfawley do we still need to tag # Experimental considering we are anyway putting under experimental folder?

Reply:
No, if anything is in an experimental package it would not require a special notice.